btc.com/test/block.test.ts

```diff
@@ -0,0 +1,88 @@
+import { assert } from 'chai'
```

Comment:
Lets use jest over chai+mocha

Reply:
This is going directly into master. No jest yet ;)